GOAT-Compiler/TypeChecker.cs

```diff
@@ -131,7 +131,7 @@ private void DivMultTypeChecker(Symbol id, Types type, Node node, Node expr, TDo
             }
             else
             {
-                throw new TypeMismatchException(node);
+                throw new TypeMismatchException(node, "Types not compatible with multiply or divide expression");
```

Comment:
"Types are not compatible with a multiply or divide expression"
eller 
"Types are not compatible with multiply or divide expressions"